readme.md

```diff
@@ -191,24 +194,88 @@ Print is the only language defined i/o function. It takes in a single variable a
 The following are valid print statements:
 ```
 print ( "hello" );
-print ( a );
+print ( 42 );
+print ( a );  # Only valid if the variable "a" is defined.
 ```
 
 ### Loops
 A loop will itterate between a range of numbers.
+
 Example loops:
 ```
-loop ( a in range, ( 1, 100 ) ) {
+loop ( a in range( 1, 100 ) ) {
 
 	print ( a );
 
 }
 ```
+### Type casting
+Most data types can be converted into any other data type.
+```
+# Convert to int.
+# Can be done with all data types.
+int(x)
+
+# Convert to float.
+# Can be done with all data types.
+float(x)
+
+# Convert to string.
+# Can be done with all data types.
+string(x)
+
+# Convert to bool.
+# Can be done with all data types.
+bool()
+
+# Convert to none.
+# Can be done with all data types.
+none(x)
+```
+
+### NOP
+Incase you need something to do nothing, you can use the `pass` keyword to do nothing.
+
+Example:
+```
+if 2 > 1 {
+
+ print ( "hello" );
+
+} 
+else { 
+
+ pass
+
+}
+```
+
+### Importing functions
+By using the `import` keyword you can use functions from other sack programs.
+
+Example:
+
+in exampleModule.sk:
+```
+func helloworld(){
+    print ( "Helloworld" );
+}
+```
+
+and in exampleProgram.sk:
+```
+import("exampleModule");
+helloworld();
+```
+
+You can only import modules that are in the current directory
```

Comment:
Simplicity is nice but you could probably be able to do `import( "some_folder/some_module" )`